robots:
  crazyflies:
    cf1:
      name: cf1
      uri: ""
      translation: "-1 -1 0.015"
      orientation: [0, 0, 0, 1]
    cf2:
      name: cf2
      uri: ""
      translation: "-2 -2 0.015"
      orientation: [0, 0, 0, 1]

  turtlebots: 
    tb1:
      name: tb1
      uri: ""
      translation: "-1.5 -1.5 0.015"
      orientation: [0, 0, 0, 1]
    tb2:
      name: tb2
      uri: ""
      translation: "-2.5 -2.5 0.015"
      orientation: [0, 0, 0, 1]

world_file_to_write_path: "apartment.wbt"

world_file_to_copy_path: "../worlds/base_worlds/apartment.wbt"